### PR TITLE
Keerthis data

### PR DESCRIPTION
**Pull request by keerthi1822**

### CODE REVIEW

**Comment by benna100** [comment]
Hey Keerthika...
Only commit what is relevant to this PR. That means your details. Not all the line spaces and reformatting!

File: components/hire/alumni.json [hunk 1]
```diff
@@ -45,24 +45,57 @@
     "onlineCV": "https://monya.netlify.com/",
     "pdfCV": "/static/alumni/cv/monya.pdf",
     "linkedin": "https://www.linkedin.com/in/monya-kwene-alete-208203b5/",
-    "skills": ["JS", "Express.js", "MySQL", "React", "HTML", "CSS", "Node.js", "Java", "Sass", "Scss", "Heroku", "Knex.js", "XML", "PostgreSQL", "GraphQL"],
+    "skills": [
```
Comment (benna100) [inline components/hire/alumni.json]: Do not add stuff like this in this PR.

File: components/hire/alumni.json [hunk 2]
```diff
@@ -159,52 +202,118 @@
...
     "graduationDate": "11-08-2019"
   },
-    {
+  {
     "id": 19,
     "roles": ["alumni"],
     "name": "Keerthika Devi Alampalli",
-    "skills": ["JS","React","Knex.js", "Express.js", "MySQL", "HTML", "CSS", "Node.js"],
+    "email": "keerthi1822@gmail.com",
+    "photo": "/static/alumni/photos/Keerthi.jpg",
```
Comment (benna100) [inline components/hire/alumni.json]: This is the information that should be there. Together with the cv and picture of you

Comment (keerthi1822) [reply components/hire/alumni.json]: Hi Benjamin, I didn't change any of the stuff that is not related to mine. I did only changes what you mentioned

Comment (benna100) [reply components/hire/alumni.json]: I know you did not do it on purpose, prettier did! But when adding, only add the lines relevant to the commit

**Pull request closed without merging**